src/Conference/ConferenceDetails.test.js

```diff
@@ -80,6 +80,7 @@ describe("ConferenceDetails", () => {
     expect(wrapper.find(".date").length).toBe(1);
     expect(wrapper.find(".time").length).toBe(1);
     expect(wrapper.find(".city").length).toBe(1);
-    expect(wrapper.find(".description").length).toBe(1);
+    expect(wrapper.find(".description").length).toBe(2);
+    //Description is length 2 because of the Styled Component used on it
```

Comment:
This is a little bit of a flag, should this not be fixed rather than leaving a comment?

Reply:
> This is a little bit of a flag, should this not be fixed rather than leaving a comment?

This element is the only one in the set of p tags to use styled components and I think that's what's causing it to register as length two since it is technically nested in the styled components tag. Is there a way to specifically assign the description tag to the p tags only and not the styled part?

Reply:
> Also no snapshot for the conference details component?

I think we stopped using them because Sethi recommended it as they break every time you change something on the page even if it still looks the same.